WalletWasabi.Fluent/Models/Wallets/TransactionTreeBuilder.cs

```diff
@@ -195,6 +197,8 @@ private TransactionModel CreateSpeedUpGroup(TransactionSummary transactionSummar
 			Labels = parent.Labels,
 			CanCancelTransaction = transactionSummary.Transaction.IsCancellable(_wallet.KeyManager),
 			CanSpeedUpTransaction = transactionSummary.Transaction.IsSpeedupable(_wallet.KeyManager),
+			Fee = transactionSummary.GetFee(),
+			FeeRate = transactionSummary.FeeRate(),
```

Comment:
This is not correct. It was just sped up.
![image](https://github.com/zkSNACKs/WalletWasabi/assets/16364053/12b0cf66-2f21-4175-8559-d69f2fff8f59)

BTW for the accelerator txn I don't even see the confirmation time:
![image](https://github.com/zkSNACKs/WalletWasabi/assets/16364053/66c6e50c-1498-4ed1-a360-7029391401e3)

Reply:
Yes, it's quite annoying, because its details already shows the est. conf time, but somehow the confirmation tool tip  doesn't get updated. I will dig deeper.